asf_search/search/baseline_search.py

```diff
@@ -84,18 +84,22 @@ def get_stack_opts(
         raise ASFBaselineError(f'Requested reference product needs a baseline stack ID but does not have one: {reference.properties["fileID"]}')
 
     # build a stack from scratch if it's a non-precalc dataset with state vectors
-    if reference.properties['platform'] in [PLATFORM.SENTINEL1A, PLATFORM.SENTINEL1B]:
-        stack_opts.platform = [PLATFORM.SENTINEL1A, PLATFORM.SENTINEL1B]
+    if reference.properties['platform'].upper() in [PLATFORM.SENTINEL1A.upper(), PLATFORM.SENTINEL1B.upper()]:
```

Comment:
Good minor addition. Double-bagging: it is expected that the next revision of the dataset will have the normal style of platform name, capitalization-wise.